token/core/zkatdlog/nogh/issuer.go

```diff
@@ -59,8 +63,16 @@ func (s *Service) Issue(issuerIdentity view.Identity, typ string, values []uint6
 	return issue, infoRaws, fid, err
 }
 
+// VerifyIssue checks if the outputs of an IssueAction match the passed tokenInfos
 func (s *Service) VerifyIssue(ia driver.IssueAction, tokenInfos [][]byte) error {
 	action := ia.(*issue.IssueAction)
+	if action == nil {
```

Comment:
can we check that ia is not nil and here that the cast was succefull?

Reply:
OK